ibm_catalog.json

```
@@ -0,0 +1,250 @@
+{
+  "products": [
+    {
+      "label": "Cloud automation for Observability agents",
+      "name": "deploy-arch-ibm-observability-agents",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "logging_monitoring",
+        "terraform",
+        "target_terraform",
+        "solution"
+      ],
+      "keywords": [
+        "observability",
+        "agents",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution"
+      ],
+      "provider_name": "IBM",
+      "short_description": "Deploys IBM Cloud Observability agents to a cluster",
+      "long_description": "Solutions that support deploying IBM Cloud Observability agents for logging, monitoring.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/solutions/baseline/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-agents/main/images/observability-icon.svg",
+      "flavors": [
+        {
+          "label": "Baseline",
+          "name": "baseline",
+          "working_directory": "solutions/baseline",
```

Also move code to this directory
```suggestion
          "working_directory": "solutions/fully-configurable",
```